### PR TITLE
Feature/ap5 14

### PR DESCRIPTION
**Pull request by jeremy-roussel**

### CODE REVIEW

File: server/src/graphql/resolvers/queries.js [hunk 1]
```diff
@@ -41,4 +41,19 @@ module.exports = {
       return true;
     } else return false;
   },
+
+  getAllRigs: async () => {
+    const AllRigs = await Rig.findAll();
```
Comment (BarretteBanner) [inline server/src/graphql/resolvers/queries.js]: camel case

File: server/src/graphql/resolvers/queries.js [hunk 2]
```diff
@@ -41,4 +41,19 @@ module.exports = {
...
+    const RigIDs = [];
+    for (rig in AllRigs) {
+      RigIDs.push(rig.id);
+    }
+
+    const AllWells = await Well.findAll({ where: { rigID: [RigIDs] } });
```
Comment (BarretteBanner) [inline server/src/graphql/resolvers/queries.js]: use the relationships to get wells

File: server/src/graphql/resolvers/mutations.js [hunk 3]
```diff
@@ -15,4 +15,17 @@ module.exports = {
     const user = await User.create({ id: uuidv4(), email, password: hash });
     return user ? true : false;
   },
+
+  deleteRig: async (_, { id }) => {
+    const isDeleted = await Rig.destroy({ where: { id } });
+    return isDeleted ? true : false;
```
Comment (mmevans) [inline server/src/graphql/resolvers/mutations.js:73]: make sure to keep the return value consistent between all delete mutations. Some of the other mutations return the entire row and other ones just the ID. I'd recommend returning the id if the deletion was successful and an error if something went wrong. Returning false to the FE isn't helpful when figuring out what went wrong.

**Review by popesicle — APPROVED** [review]

File: server/src/graphql/resolvers/mutations.js (hunk 3 above)
Comment (BarretteBanner) [reply server/src/graphql/resolvers/mutations.js:73]: ☝️

**Comment by BarretteBanner** [comment]
conflicts

**Review by BarretteBanner — APPROVED** [review]

**Pull request merged**